### PR TITLE
Issue 34

### PR DESCRIPTION
**Pull request by Friedjof**
Please read the documentation and maybe clone the repo again for a test from scratch.

### CODE REVIEW

**Comment by Friedjof** [comment]
Wait...

**Review by felbinger — COMMENTED** [review]
Why?
Would be e,g, `helpwave-backend_postgres_1` after running `git clone helpwave-backend` and `docker compose up -d`, which is the default schema (`project_definition-name_instance`) for docker compose.
When adjusting the container_name, you loose the ability to handle multiple instances of this container.
The hostname should be used instead: `helpwave-backend_postgres_1`

**Comment by felbinger** [comment]
Also if you're still working on it, I suggest creating a draft pr, so people see that the review is not required yet.

**Comment by Friedjof** [comment]
done.

File: app/requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ psycopg2-binary==2.9.3
 SQLAlchemy==1.4.41
 uvicorn==0.18.3
 passlib==1.7.4
-psycopg2==2.9.3
```
Comment (felbinger) [inline app/requirements.txt:6]: @Friedjof ist das gewollt?

**Review by felbinger — APPROVED** [review]
Why do you remov

**Pull request merged**